### PR TITLE
added questions on validation section

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: data/validation.yml [hunk 1]
```diff
@@ -19,3 +19,15 @@ questions:
         answers:
             - {value: "True",    correct: true}
             - {value: "False",   correct: false}
+    -
+        question: 'True or False ? We can also apply constraints on class getters with ``addPropertyConstrait()``'
+        answers:
+            - {value: "True",    correct: false}
+            - {value: "False",   correct: true}
+    -
+        question: 'True or False ? In Symfony:'
```
Comment (supertanuki) [inline data/validation.yml]: I rather propose: "In Symfony, which one of these sentence is correct?"

Comment (mickaelandrieu) [inline data/validation.yml]: why not ? I will update my pull request. Thank you for your interest, don't be shy, propose some new questions :)

File: data/validation.yml [hunk 2]
```diff
@@ -19,3 +19,15 @@ questions:
...
+        answers:
+            - {value: "Annotations mapping are enabled by default",                correct: false}
+            - {value: "UniqueEntity is provided by Doctrine Bundle",               correct: false}
+            - {value: "We can validate partial object with @Assert\GroupSequence", correct: true}
```
Comment (xabbuh) [inline data/validation.yml]: I think this is misleading as you don't need a group sequence to apply only a subset of all validation rules.

Comment (mickaelandrieu) [inline data/validation.yml]: How about "We can priorize a set of validation rules with @Assert\GroupSequence" ?

Comment (xabbuh) [inline data/validation.yml]: Not sure that's better. Prioritize sounds like we only control the order.

Comment (mickaelandrieu) [inline data/validation.yml]: @xabbuh yeah, it's hard to explain in few words ... GroupSequence is more to add new validation rules to an objectt if this object already validate rules.

**Pull request merged**